src/envoy/auth/http_filter.cc

```diff
@@ -51,5 +115,90 @@ void JwtVerificationFilter::setDecoderFilterCallbacks(
   decoder_callbacks_ = &callbacks;
 }
 
+void JwtVerificationFilter::ReceivePubkey(HeaderMap& headers,
+                                          std::string issuer_name, bool succeed,
+                                          const std::string& pubkey) {
+  auto iss_it = calling_issuers_.find(issuer_name);
```

Comment:
do we handle case where async call has been timeout,  and client request object has been destroyed.

Reply:
fixed